prijateli_tree/app/main.py

```diff
@@ -14,3 +28,60 @@
 @app.get("/")
 def funky():
     return {"Hello": "World"}
+
+
+@app.get("/administration")
+def admin_access():
+    stuff()
+
+
+@app.post("/create-game/")
+def create_game_endpoint(game_type: int, user_id: int, num_rounds: int, practice: bool):
+    try:
+        new_game_id = create_new_game(game_type, user_id, num_rounds, practice)
+        return {"status": "success", "game_id": new_game_id}
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
+
+
+@app.post("/game/{game_id}/add-player/")
```

Comment:
This should be taken care of somewhere else, particularly the admin panel. Games are set up with a particular set of players in them.